Index: Assets/GameObjects/SoundsAndMusic.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 3425d5da8f8e7ec4c93bf56a72ad64b0\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/SoundsAndMusic.prefab.meta b/Assets/GameObjects/SoundsAndMusic.prefab.meta
--- a/Assets/GameObjects/SoundsAndMusic.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/SoundsAndMusic.prefab.meta	(date 1717336245422)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 3425d5da8f8e7ec4c93bf56a72ad64b0
+guid: 318cd6f48e6d7c645b2daa3b1fe672f5
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/BlurManager.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 5ba1189c58402854c955e128ff1e6938\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/BlurManager.prefab.meta b/Assets/GameObjects/BlurManager.prefab.meta
--- a/Assets/GameObjects/BlurManager.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/BlurManager.prefab.meta	(date 1717336245413)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 5ba1189c58402854c955e128ff1e6938
+guid: 90d2c8e9d3984a9498d45bb34b7ea2df
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/SceneAnimations.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: fc507f5519e9f904b8aa74516fb71efa\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/SceneAnimations.prefab.meta b/Assets/GameObjects/SceneAnimations.prefab.meta
--- a/Assets/GameObjects/SceneAnimations.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/SceneAnimations.prefab.meta	(date 1717336245419)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: fc507f5519e9f904b8aa74516fb71efa
+guid: f91f0e09fc0c54d43893d4047d68ec5d
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/PauseMenu.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: fa7f2dc53d192794394e7356517efa67\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/PauseMenu.prefab.meta b/Assets/GameObjects/PauseMenu.prefab.meta
--- a/Assets/GameObjects/PauseMenu.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/PauseMenu.prefab.meta	(date 1717336245416)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: fa7f2dc53d192794394e7356517efa67
+guid: 8d4de3c4532c2784483dbfebc8253e92
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/PipeTemplate.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 83151f4b7def6874ebd00b7c34df946c\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/PipeTemplate.prefab.meta b/Assets/GameObjects/PipeTemplate.prefab.meta
--- a/Assets/GameObjects/PipeTemplate.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/PipeTemplate.prefab.meta	(date 1717336245417)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 83151f4b7def6874ebd00b7c34df946c
+guid: 03a4887f3ad242547aaf943a42814281
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/CameraShake.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: b332517f845c7f64eb458502251a1982\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/CameraShake.prefab.meta b/Assets/GameObjects/CameraShake.prefab.meta
--- a/Assets/GameObjects/CameraShake.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/CameraShake.prefab.meta	(date 1717336245413)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: b332517f845c7f64eb458502251a1982
+guid: 7791512239c118446ba8f65fc78494d2
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/background.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 4256f71dbf7f558419516492d99339a0\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/background.prefab.meta b/Assets/GameObjects/background.prefab.meta
--- a/Assets/GameObjects/background.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/background.prefab.meta	(date 1717336245412)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 4256f71dbf7f558419516492d99339a0
+guid: 7798eb6b9132cb740a2ccdebfe1dc131
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/SoundMenuPrefab.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 47aec839cf5dbc74198c036bad822593\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/SoundMenuPrefab.prefab.meta b/Assets/GameObjects/SoundMenuPrefab.prefab.meta
--- a/Assets/GameObjects/SoundMenuPrefab.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/SoundMenuPrefab.prefab.meta	(date 1717336245421)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 47aec839cf5dbc74198c036bad822593
+guid: dca58bf9a7b701a4ba1274b792533d79
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/river pref.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 1c6a5b74ead125348adac112bf034620\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/river pref.prefab.meta b/Assets/GameObjects/river pref.prefab.meta
--- a/Assets/GameObjects/river pref.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/river pref.prefab.meta	(date 1717336245418)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 1c6a5b74ead125348adac112bf034620
+guid: e8eba0a8fe4935046a827de4a1c8e486
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/GroundTiles.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: e5b64dc0d7c8638468ee67455eaec4af\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/GroundTiles.prefab.meta b/Assets/GameObjects/GroundTiles.prefab.meta
--- a/Assets/GameObjects/GroundTiles.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/GroundTiles.prefab.meta	(date 1717336245415)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: e5b64dc0d7c8638468ee67455eaec4af
+guid: 5129d9682ef71d84da1f947d6b71db60
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/Finish.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 2b21baf6e116da946a362eeefa9437c3\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/Finish.prefab.meta b/Assets/GameObjects/Finish.prefab.meta
--- a/Assets/GameObjects/Finish.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/Finish.prefab.meta	(date 1717336245414)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 2b21baf6e116da946a362eeefa9437c3
+guid: 833ab00c122294f4582af0c095b3b66b
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/newGroundTiles.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 56ce7b7323358d64497bf69b718bc318\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/newGroundTiles.prefab.meta b/Assets/GameObjects/newGroundTiles.prefab.meta
--- a/Assets/GameObjects/newGroundTiles.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/newGroundTiles.prefab.meta	(date 1717336245416)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 56ce7b7323358d64497bf69b718bc318
+guid: 13e61dcab05f70748a338b941087d16a
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/Player.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 9804d15014e8c744eba9f5f9c6f026d8\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/Player.prefab.meta b/Assets/GameObjects/Player.prefab.meta
--- a/Assets/GameObjects/Player.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/Player.prefab.meta	(date 1717336245418)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 9804d15014e8c744eba9f5f9c6f026d8
+guid: 4e2d0b4a2b42c89429e21f7ba7a249fc
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/Main Camera.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 91a0a12afa97cdf4baeeb962bcf68ed3\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/Main Camera.prefab.meta b/Assets/GameObjects/Main Camera.prefab.meta
--- a/Assets/GameObjects/Main Camera.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/Main Camera.prefab.meta	(date 1717336245415)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 91a0a12afa97cdf4baeeb962bcf68ed3
+guid: 3342ef4cdaf48af409e636c68dae4ae3
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/SceneManager.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 945845dc801c2cd4d99a0a472c00034b\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/SceneManager.prefab.meta b/Assets/GameObjects/SceneManager.prefab.meta
--- a/Assets/GameObjects/SceneManager.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/SceneManager.prefab.meta	(date 1717336245420)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 945845dc801c2cd4d99a0a472c00034b
+guid: cea30f0f816347b4cb642ebd5546962c
 PrefabImporter:
   externalObjects: {}
   userData: 
Index: Assets/GameObjects/SoundFXObject.prefab.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>fileFormatVersion: 2\r\nguid: 65141a4476c03354b819d903d4315aa5\r\nPrefabImporter:\r\n  externalObjects: {}\r\n  userData: \r\n  assetBundleName: \r\n  assetBundleVariant: \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/GameObjects/SoundFXObject.prefab.meta b/Assets/GameObjects/SoundFXObject.prefab.meta
--- a/Assets/GameObjects/SoundFXObject.prefab.meta	(revision 3bfc24aee08b3812cb653d331a8d4da449229df0)
+++ b/Assets/GameObjects/SoundFXObject.prefab.meta	(date 1717336245421)
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 65141a4476c03354b819d903d4315aa5
+guid: 8707078318d6fdc478b57439066876af
 PrefabImporter:
   externalObjects: {}
   userData: 
